src/hardware/voodoo.cpp

```diff
@@ -178,16 +182,15 @@ PageHandler* VOODOO_GetPageHandler() {
     return nullptr;
 }
 
-void VOODOO_Destroy(Section* sec) {
+void VOODOO_Destroy(Section* /*sec*/) {
```

Comment:
`[[ maybe_unused ]]`

Reply:
This is so GCC (Meson) doesn't give a warning... @Wengier code also have the same thing

Reply:
@kcgen meant to literally put in `[[ maybe_unused ]]` there 😄 Grep the codebase for examples.

Reply:
It's perfectly fine to comment out the parameter. If you want the parameter there anyway, you can also just (void)param; right below it. That counts as "using" the parameter and silences that warning.